frontend/projects/src/app/components/form/TextBoxArea/TextBoxArea.tsx

```diff
@@ -0,0 +1,19 @@
+import style from "./TextBoxArea.module.css";
+
+type TextBoxAreaProps = {
+    textBoxAreaName: string;
+    textBoxAreaLabel: string;
+    textBoxAreaRows: number;
+    textBoxAreaCols: number;
```

Comment:
任意の引数にしましょう
指定しない場合のデフォルト値を決定してあげましょう

```suggestion
    textBoxAreaRows?: number;
    textBoxAreaCols?: number;
```